src/Stratis.Bitcoin.Features.Miner/PowMining.cs

```diff
@@ -29,12 +29,18 @@ public class PowMining
         // Default for -blockmintxfee, which sets the minimum feerate for a transaction in blocks created by mining code.
         public const int DefaultBlockMinTxFee = 1000;
 
-        // Default for -blockmaxsize, which controls the maximum size of block the mining code will create.
+        // Default for -blockmaxsize, which controls the maximum number of serialized bytes in a block the mining code will create.
         public const int DefaultBlockMaxSize = 750000;
 
-        // Default for -blockmaxweight, which controls the range of block weights the mining code will create.
+        /// <summary>
+        /// Default for -blockmaxweight, which controls the maximum of block weight the mining code will create.
+        /// Block is measured in weight units. Data which touches the UTXO (What addresses are involved in the transaction, how many coins are being transferred) costs
+        /// 4 weight units (WU) per byte. Witness data (signatures used to unlock existing coins so that they can be spent) costs 1 WU per byte.
+        /// <remarks>See also: http://learnmeabitcoin.com/faq/segregated-witness </remarks>
+        /// </summary>
         public const int DefaultBlockMaxWeight = 3000000;
 
+        /// <summary>InnerLoopCount = 65536.</summary>
```

Comment:
that's not a good description of the constant
also we are missing an access modifier here, can we add it pls?